portal/templates/profile/profile_macros.html

```diff
@@ -253,33 +253,43 @@ <h4 class="{{prefix}}-label text-warning">{{label}}</h4>
-                {% endcall %}
-            {% endif %}
-            <button id="btnProfileSendEmail" class="btn btn-tnth-primary disabled" type="button">{{ _("Send email") }}</button>
-            <div id="profileEmailMessage" class="text-info"></div>
-            <div id="profileEmailErrorMessage" class="text-danger"></div>
+{% macro profileEmailForm(prefix='registration', status_title=_('Registration Status:'), status_label=_('not registered'), class='') -%}
+    <div class="form-group well profile-email-container {{class}}" id="profile{{prefix}}SendEmailContainer" data-profile-section-id="patientemailform" data-email-type="{{prefix}}">
+       <div id="{{prefix}}StatusContainer" class="{{prefix}}-status-container">
+            <span class="text-muted">{{status_title}}</span>
+            <h4 class="{{prefix}}-label text-warning">{{status_label}}</h4>
+            <div class="email-selector-container">
+                <label id="send{{prefix}}EmailLabel" class="text-muted">{{_("Send email to patient")}}</label>
+                <select id="profile{{prefix}}EmailSelect" class="form-control bd-element {{prefix}}-email-selector email-selector" aria-labelledby="sendEmailLabel">
+                    <option value="">{{_("Select Email...")}}</option>
+                    {{caller()}}
+                </select>
+            </div>
         </div>
-    {%- endif -%}
+        <button id="btnProfileSend{{prefix}}Email" class="btn-send-email btn btn-tnth-primary disabled" type="button">{{ _("Send email") }}</button>
+        <div id="profile{{prefix}}EmailMessage" class="send-email-info text-info"></div>
+        <div id="profile{{prefix}}EmailErrorMessage" class="send-email-error text-danger"></div>
+    </div>
 {%- endmacro %}
 {% macro profileSendEmail(person) -%}
     {%- if person -%}
-        <div class="form-group float-input-label well well-lg" id="profileSendEmailContainer">{{profileEmailForm(person)}}</div>
-    {% endif %}
+        <div class="email-form-container">
+            {% call profileEmailForm() %}
+                {%if person.has_role('write_only')%}
+                    <option value="invite" data-url="{{url_for('portal.patient_invite_email', user_id=person.id)}}">{{_("TrueNTH Invite")}}</option>
+                {%endif %}
+                <option value="reminder" data-url="{{url_for('portal.patient_reminder_email', user_id=person.id)}}">{{_("TrueNTH Reminder")}}</option>
+            {% endcall %}
+            <!--
+                set P3P invite/reminder tile visibility based on whether user's org matches config variable, ACCEPT_TERMS_ON_NEXT_ORG - this is the case of MUSIC patient
+            -->
+            {% set cssClass %}{%- if person.first_top_organization() and (person.first_top_organization().name == config.ACCEPT_TERMS_ON_NEXT_ORG) -%}active{%-endif-%}{% endset %}
+            {% call profileEmailForm(prefix='assessment', status_title=_('P3P Assessment Status:'), status_label=_('not available'), class=cssClass) %}
```

Comment:
Just a heads up- these strings are specific to TNUSA, but will also be extracted and translated for ePROMs. There's not many but we may need to figure out a way to have profile macros in both template directories